src/components/Footer/Footer.js

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React, { Component } from 'react';
+import * as PropTypes from 'prop-types';
+import { uuid4 } from '../../utils/utils';
+import {
+  BUTTON_CLASSES, DROPDOWN_CLASSES, FOOTER_CLASSES, GENERIC_SIZES, ICON_CLASS, LIGHT_CLASS, UTILITY_CLASSES,
+} from '../../constants/classes';
+
+// eslint-disable-next-line react/prop-types
+function LanguageDropdown({ isMobile, buttonId }) {
+  const languages = [
+    'English',
+    'Español',
+    'Português',
+    'Français',
+    'Deutsch',
+    '简体中文',
+    '日本語',
+  ];
+  const footerLanguageClass = isMobile
+    ? ['-justify-content--center -w--100 -pr--0 -pb--2']
+    : [];
+
+  return (
+    <div
+      className={`${DROPDOWN_CLASSES.DROPDOWN} ${FOOTER_CLASSES.FOOTER_LANGUAGE} ${footerLanguageClass}`}>
+      <a
+        className={`${BUTTON_CLASSES.BUTTON} ${BUTTON_CLASSES.ICON_BUTTON} ${BUTTON_CLASSES.FLAT} ${LIGHT_CLASS} ${GENERIC_SIZES.SM} ${DROPDOWN_CLASSES.TRIGGER}`}
+        id={buttonId}
+        data-position="top-start"
+        aria-label="Select your preferred language">
+        <div className={BUTTON_CLASSES.CONTENT}>
+          <i
+            className={`${ICON_CLASS} icon-globe-network-outline`}
+            aria-hidden="true">
+          </i>
+          <span>English</span>
+        </div>
+      </a>
+      <div className={`${DROPDOWN_CLASSES.MENU} -w--sm ${UTILITY_CLASSES.TYPOGRAPHY.TEXT_BODY}`}>
+        {languages.map((lang) => (
+          <a
+            key={lang}
+            className={`${DROPDOWN_CLASSES.ITEM} ${lang === 'English' ? '-active' : ''}`}
+            href="#">
+            {lang}
+          </a>
+        ))}
+      </div>
+    </div>
+  );
+}
+
+export default class Footer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { id: uuid4() };
+  }
+
+  componentDidMount() {
+    const buttonId = `${this.state.id}-language-dropdown-button`;
+    const initialize = setInterval(() => {
+      if (window.chi && document.getElementById(buttonId)) {
+        window.chi.dropdown(document.getElementById(buttonId));
+        clearInterval(initialize);
+      }
+    }, 1000);
+  }
+
+  render() {
+    const { id } = this.state;
+    const buttonId = `${id}-language-dropdown-button`;
+    const isTabletSize = this.props.footerSize === 'Portrait tablet';
+    const isDesktopSize = this.props.footerSize === 'Desktop';
+    const footerStyle = isDesktopSize
+      ? { minWidth: '1256px' }
+      : { width: isTabletSize ? '736px' : '368px' };
+    const footerInternalContentStyle = isDesktopSize ? '-flex--row' : '-flex--column';
+    const ulElementStyle = isDesktopSize ? '-flex--row' : '-flex--column';
```

Comment:
We can use only one constant for both cases as they have the same value. Include `-flex--row` and column in the **classes.js** and use it here.

```suggestion
    const footerInternalContentStyle = isDesktopSize ? '-flex--row' : '-flex--column';
    const ulElementStyle = isDesktopSize ? '-flex--row' : '-flex--column';
```